server/channels/app/export_test.go

```diff
@@ -1144,6 +1144,77 @@ func TestExportDeletedTeams(t *testing.T) {
 	}
 }
 
+func TestExportSingleTeam(t *testing.T) {
+	th1 := Setup(t).InitBasic()
+	defer th1.TearDown()
+
+	teams1, errTeams := th1.App.GetAllTeams()
+	assert.Nil(t, errTeams)
```

Comment:
Why are all of these unique error names needed? Can we just use `err` everywhere?